week-1/8-advanced-selectors/styles.css

```diff
@@ -1 +1,11 @@
 /* Try different box model properties below */
+.side-header > p {
+  color: white;
+  margin-bottom: 8;
+}
+.site footer > p {
```

Comment:
This looks like a mistake - it should be `.site-footer`. Always test your changes before committing.